src/Modules/Talents/components/TalentsDetailsDrawer/Experience.tsx

```diff
@@ -0,0 +1,11 @@
+import { Box } from "@chakra-ui/react";
+
+export const Experience = (): JSX.Element => {
+  return (
+    <>
```

Comment:
remove unneccrery empty tags, to decrease the DOM size
please read this :https://nitropack.io/blog/post/avoid-an-excessive-dom-size